components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -582,4 +583,30 @@ - (void)updateDisplayedTitle {
   _titleLabel.text = [[self class] displayedTitleForTitle:_title style:_style];
 }
 
+- (CGSize)badgeLabelSizeWithText:(NSString *)string font:(UIFont *)font {
+  if (string.length <= 0) {
+    return CGSizeZero;
+  }
+
+  NSMutableString *longestAllowableBadgeString = [[NSMutableString alloc] init];
+  __block NSUInteger composedCharacterIndex = 0;
+  [string enumerateSubstringsInRange:NSMakeRange(0, string.length)
+                             options:NSStringEnumerationByComposedCharacterSequences
+                          usingBlock:^(NSString *substring, NSRange substringRange,
+                                       NSRange enclosingRange, BOOL *stop) {
+                            [longestAllowableBadgeString appendString:substring];
+                            composedCharacterIndex++;
+                            if (composedCharacterIndex == kBadgeMaxTextComposedCharacterLength) {
```

Comment:
This was supposed to be changed to >=

Reply:
I don't think it's necessary, since we're saying `*stop = YES;` a line below

Reply:
Yes, but if by some stroke of luck 2 characters are accumulated, it would go on forever. That's why I originally asked for `>=`.  You also said you would change it before submitting.

> The *stop = YES; line tells it to terminate, but I can do a >= too

Reply:
I think I'm missing something here--what value of `string` would result in this going on forever?

Whoops! Must have slipped me by. Apologies Rob. I'll put up a PR with the change.